rslang/src/scss/components/_notification.scss

```diff
@@ -1,21 +1,27 @@
 .notification-root {
   .MuiSnackbarContent-message {
     color: black;
-    @include font(
-      $line-height: 130%,
-      $spacing: 0.02rem
-    );
+    @include font($line-height: 130%, $spacing: 0.02rem);
   }
 
-  .SnackbarItem-variantSuccess-20 {
-    background-color: $color-success;
+  &.notification_success {
+    .MuiSnackbarContent-root,
+    .MuiPaper-root {
+      background-color: $color-success;
+    }
   }
 
-  .SnackbarItem-variantError-21 {
-    background-color: $color-error;
+  &.notification_error {
+    .MuiSnackbarContent-root,
+    .MuiPaper-root {
+      background-color: $color-error;
+    }
```

Comment:
Please, don't change general styles. If you want, you can add your class to notification and then change styles by your own selector